rule/eq_expr.go

```diff
@@ -62,3 +68,152 @@ func (n *exprEq) Eval(params Params) (*Value, error) {
 
 	return BoolValue(true), nil
 }
+
+/////////////////
+// LT operator //
+/////////////////
+
+type exprLT struct {
+	operator
+}
+
+func newExprLT() *exprLT {
+	return &exprLT{
+		operator: operator{
+			contract: Contract{
+				OpCode:     "lt",
+				ReturnType: BOOLEAN,
+				Terms: []Term{
+					{
+						Type:        ANY,
+						Cardinality: MANY,
+						Min:         2,
+					},
+				},
+			},
+		},
+	}
+}
+
+// LT creates an expression that takes at least two operands and
+// evaluates to true if each successive operand has a lower value than
+// the next.
+func LT(vN ...Expr) Expr {
+	e := newExprLT()
+	e.consumeOperands(vN...)
+	return e
+}
+
+// perform a less-than comparison on a sequence of integers
+func (n *exprLT) integerLT(params Params) (*Value, error) {
+	var i0, i1 int64
+	var err error
+
+	i0, err = exprToInt64(n.operands[0], params)
+	if err != nil {
+		return nil, err
+	}
+	for j := 1; j < len(n.operands); j++ {
+		i1, err = exprToInt64(n.operands[j], params)
+		if err != nil {
+			return nil, err
+		}
+		if i0 >= i1 {
```

Comment:
Given that you are implementing the LT symbol, the check should be `>` instead of `>=`, isn't it?
And it will be `>=` for the LTE one.

Reply:
Not that the next line returns `BoolValue(false)` - It's `>=` here because the only true case is `<`.

Reply:
Oh right, my bad!